Ginger/GingerCore/Drivers/Appium/SeleniumAppiumDriver.cs

```diff
@@ -1458,6 +1458,17 @@ bool IWindowExplorer.AddSwitchWindowAction(string Title)
         }
 
         public event Amdocs.Ginger.Plugin.Core.RecordingEventHandler RecordingEvent;
+        object Amdocs.Ginger.Plugin.Core.IRecord.CurrentBusinessFlow
```

Comment:
why adding CurrentBusinessFlow to interface is needed?

Reply:
while initial implementation i was trying to check if the businessflow is changed or not so i added this property, which might be used in future